crates/wasmtime/src/store.rs

```diff
@@ -127,6 +131,46 @@ impl Store {
         }
     }
 
+    /// Gets a host function from the [`crate::Config`] associated with this [`Store`].
+    ///
+    /// Returns `None` if the given host function is not defined.
+    pub fn get_host_func(&self, module: &str, name: &str) -> Option<Func> {
+        self.inner
+            .engine
+            .config()
+            .get_host_func(module, name)
+            .map(|f| f.to_func(self))
```

Comment:
Seems like, based on this call site, they should stay in-sync, but that perhaps `to_func` isn't always guaranteed to be safe. If I'm understanding all this correctly, then I think `to_func` should be `unsafe` and this call site should be the place with the `unsafe` block that has a comment like "we know this store is using the same config as the host func".

Reply:
I'll make it unsafe and comment as to the safety of the call.